composer.json

```diff
@@ -26,7 +26,7 @@
     "source": "https://github.com/cakephp/chronos"
   },
   "require": {
-    "php": ">=5.5.9"
+    "php": "^5.5.9|^7"
```

Comment:
You should use a double pipe

https://getcomposer.org/doc/articles/versions.md#writing-basic-version-constraints